src/ui/config/config.go

```diff
@@ -183,6 +182,11 @@ func LDAP() (*models.LDAP, error) {
 	ldap.Filter = cfg[common.LDAPFilter].(string)
 	ldap.Scope = int(cfg[common.LDAPScope].(float64))
 	ldap.Timeout = int(cfg[common.LDAPTimeout].(float64))
+	if cfg[common.LDAPVerifyCert] != nil {
+		ldap.VerifyCert = cfg[common.LDAPVerifyCert].(bool)
+	} else {
+		ldap.VerifyCert = false
```

Comment:
this is not necessary by default a boolean member will be false.

Reply:
Updated